### PR TITLE
Gftools packager ofl kodemono

### PR DESCRIPTION
**Pull request by isaozler**

### CODE REVIEW

**Comment by emmamarichal** [comment]
Hello @isaozler, thank you for that PR! 
I noticed some interpolation issues here:

Screenshot 2024-01-26 at 11 55 15
Screenshot 2024-01-26 at 11 55 46

Also, the url for the OFL license changed recently, it's now `https://openfontlicense.org/` instead of `https://scripts.sil.org/OFL`
It needs to be changed in the OFL.txt and in the font infos (License + License URL)

Thank you :)

File: ofl/kodemono/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<p>
+    A custom-designed typeface explicitly created for the developer community.
+</p>
+<p>
+    This typeface is designed to enhance the user experience and reflect our principles of functionality and
+    timelessness.
+</p>
+<p>
+    To contribute, see <a href="https://github.com/isaozler/kode-mono">github.com/isaozler/kode-mono</a>.
+</p>
```
Comment (emmamarichal) [inline ofl/kodemono/DESCRIPTION.en_us.html:10]: > WARN The last characther on DESCRIPTION.en_us.html is not a                
          line-break. Please add it. [code: missing-eof-linebreak]   

We need to let a line-break at the end of the description file, same for metadata.pb file :)

File: ofl/kodemono/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Kode Mono Project Authors (https://github.com/isaozler/kode-mono)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```
Comment (emmamarichal) [inline ofl/kodemono/OFL.txt:5]: OFL url is no longer "https://scripts.sil.org/OFL". Use 'https://openfontlicense.org/' instead.

File: ofl/kodemono/METADATA.pb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+subsets: "menu"
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/isaozler/kode-mono"
+  commit: "15b36a40179d3678e0fe8bff19903af09c0fc0f7"
+}
```
Comment (emmamarichal) [inline ofl/kodemono/METADATA.pb:27]: Add at the end (and let a line break after stroke:"SANS_SERIF")
classification: "MONOSPACE"
stroke: "SANS_SERIF"

**Comment by emmamarichal** [comment]
@isaozler If you could update your repo, it would be great! You can read this: https://googlefonts.github.io/gf-guide/upstream.html

File: ofl/kodemono/DESCRIPTION.en_us.html (hunk 1 above)
Comment (isaozler) [reply ofl/kodemono/DESCRIPTION.en_us.html]: Resolved with this commit https://github.com/google/fonts/pull/7136/commits/e753e75e8ecc8aba2feb8a84dadda50fc45ee403

**Comment by isaozler** [comment]
Hi @emmamarichal, fixed the issues 👍

**Comment by emmamarichal** [comment]
Hi @isaozler! Thanks a lot :)

Sorry, I noticed some other small issues to fix, after that, it should be good :)
- In font info: 
    - LicenseURL = `https://openfontlicense.org` instead of `https://scripts.sil.org/OFL`
    - License = `This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: https://openfontlicense.org` (URL is modified too)
- The slash in lslash, should be rotated a bit, the glyphs seems too dark otherwise.
- Problem with ogonek
- Small figures looks very light compare to the base letters. Should be a little darker. (for this point, it's more an advice than a request)
Screenshot 2024-01-31 at 16 18 21
Screenshot 2024-01-31 at 16 16 15
Screenshot 2024-01-31 at 16 17 57


I just have to fails lefts, I'll check this with my colleagues.

<div><details><summary>☠ <b>FATAL:</b> Check METADATA.pb parse correctly. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/parses">com.google.fonts/check/metadata/parses</a>)</summary><div>
* ☠ **FATAL** Family metadata at . failed to parse.
TRACEBACK:
28:1 : Message type "google.fonts_public.FamilyProto" has no field named "classification". [code: parsing-error]
</div></details><details><summary>🔥 <b>FAIL:</b> Check for codepoints not covered by METADATA subsets. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/unreachable_subsetting">com.google.fonts/check/metadata/unreachable_subsetting</a>)</summary><div>


* 🔥 **FAIL** Could not parse metadata.pb file [code: unparsable-metadata]
</div></details>

**Comment by emmamarichal** [comment]
@isaozler I'll add your designer profile to the catalog :) Also, if you have some nice images for promotion on social media, feel free to send them to me by email! Thanks!

**Comment by isaozler** [comment]
Hi @emmamarichal, thanks for your reply. I updated/fixed the issues mentioned. The last point/advice was something that was bothering me too. So, what I did to fix that for now is to use the bold variant for all instances. I am afraid that going bolder on the bold one will not work great. I will update the number glyphs in the next iteration to be specific for the inferior and superior. But for now, we can stick to using this variant.

**Comment by isaozler** [comment]
I'm looking forward to your review 👍 I'll reach out to you concerning the promotional material for social media. Thanks in advance.

**Comment by emmamarichal** [comment]
Hi @isaozler!
That's much better, thank you :) 

I caught last few things, since you will probably correct the 8, I put here some other suggestions for improving:
- Check ogonek alignments
- The commaaccent and hungarumlaut look a bit thin in the regular master
- The eight has a different weight in fractions
- The returned comma accent looks a bit heavy and tall, and too close from the 'g'

Screenshot 2024-02-01 at 10 12 23
Screenshot 2024-02-01 at 10 12 45
Screenshot 2024-02-01 at 10 12 51
Screenshot 2024-02-01 at 10 12 18

Screenshot 2024-02-01 at 10 14 22
Screenshot 2024-02-01 at 10 14 58

Screenshot 2024-02-01 at 10 11 43

Screenshot 2024-02-01 at 10 14 39

**Comment by isaozler** [comment]
🙏 Great catches 💯 thanks. Fixed the issues and suggestions for improvement.

**Review by emmamarichal — APPROVED** [review]

**Comment by emmamarichal** [comment]
Excellent @isaozler, thanks a lot!

**Pull request merged**